src/app/features/tech-record/components/tech-record-amend-vrm/tech-record-amend-vrm.component.ts

```diff
@@ -139,4 +158,6 @@ export class AmendVrmComponent implements OnInit, OnChanges {
   setReasonForCreation(vehicleRecord: VehicleTechRecordModel) {
     if (vehicleRecord.techRecord !== undefined) vehicleRecord.techRecord.forEach(record => (record.reasonForCreation = `Amending VRM.`));
   }
+
+  toggleCherishedTransfer() {}
```

Comment:
?

Reply:
Leftover stuff from when I was trying to understand the checkboxes :p